spec/calculate_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'calculate'
+describe Calculate_Order do
+
+  let (:including_class){ Class.new { include Calculate_Order} }
+
+  test_menu = {:Hamburger => 4.50, :Chips => 1}
+  test_order = [[:Hamburger, :Chips],[1,2]]
+  test_order_list = [[:Hamburger, :Chips],[1,2],[6.50]]
+
+  describe '#check_calculation'do
+    it 'checks to see whether the expected value is the same as the calculated'do
+      expect(including_class.new.check_calculation(test_order_list, test_menu)).to eq true
```

Comment:
Metrics/LineLength: Line is too long. [90/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)